src/test/scala/nl.knaw.dans.easy.deposit/docs/JsonUtilSpec.scala

```diff
@@ -33,15 +36,21 @@ class JsonUtilSpec extends TestSupportFixture {
       typeOf[DatasetMetadata],
       typeOf[DepositInfo],
       typeOf[StateInfo],
-      typeOf[UserInfo]
+      typeOf[UserInfo],
+      typeOf[FileInfo]
```

Comment:
just to make the next diff smaller

```suggestion
      typeOf[FileInfo],
```